src/component/timeBlock/template/TimeBox.tsx

```diff
@@ -0,0 +1,79 @@
+import { styled } from 'styled-components';
+import { ErrorMessage } from '@/component/timeBlock/atoms/ErrorMessage';
+import {
+  TimeBlockGroup,
+  TimeList,
+  SelectAllButton,
+} from '@/component/timeBlock/molecules';
+import { TimeBlockHeader } from '@/component/timeBlock/organisms';
+import { useState } from 'react';
+
+export const TimeBox = () => {
+  const [nowPage, setNowPage] = useState(1);
+  const [activeDate, setActiveDate] = useState<boolean[]>([]);
+
+  return (
+    <TimeBoxContainer>
+      <TimeBlockHeader
+        page={nowPage}
+        setPage={setNowPage}
+        activeDate={activeDate}
+      />
+      <TimeBoxBlock>
+        <TimeBoxContent>
+          <TimeList start="9:00" end="17:00" />
+          <TimeBlockGroup page={nowPage} onSetActiveDate={setActiveDate} />
+        </TimeBoxContent>
+      </TimeBoxBlock>
+      {activeDate.includes(false) ? (
+        <ErrorMessage>시간이 선택되지 않은 날짜가 있습니다</ErrorMessage>
+      ) : null}
+      <SelectAllButton>모든 시간 선택하기</SelectAllButton>
+    </TimeBoxContainer>
+  );
+};
+
+const TimeBoxContainer = styled.div`
+  margin: 0 20px;
+`;
+
+const TimeBoxBlock = styled.div`
+  aspect-ratio: 1/0.8;
+  display: flex;
+  justify-content: center;
+  border-radius: 0 0 16px 16px;
+  min-width: 320px;
+  max-width: 500px;
+  min-height: 256px;
+  max-height: 424px;
+  border-left: 1px solid ${({ theme }) => theme.colors.gray5};
+  border-right: 1px solid ${({ theme }) => theme.colors.gray5};
+  border-bottom: 1px solid ${({ theme }) => theme.colors.gray5};
```

Comment:
요건 강력추천은 아니고 이런 방법도 있어요~ 같은 느낌으로

```suggestion
  border: 1px solid ${({ theme }) => theme.colors.gray5};
  border-top: 0;
```